### PR TITLE
feat: project card + tailwind

### PR DESCRIPTION
**Pull request by rohilpinto**
# Description

Removed vanilla extract styles & changed card theme to black with layout changes

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Showcase

Please attach a picture or a video showing what this change does.

image

### CODE REVIEW

**Comment by ivanms1** [comment]
Task linked: [CU-35yg2cn project card css redesign](https://app.clickup.com/t/35yg2cn)

File: apps/web/src/components/ProjectCard/ProjectCard.tsx [hunk 1]
```diff
@@ -77,53 +65,58 @@ const ProjectCard = ({ project }: ProjectCardProps) => {
...
+      <div className='flex cursor-pointer'>
+        <Link
+          href={{
+            pathname: `/project/${project.id}`,
+          }}
+          passHref
+        >
           <Image
-            className={previewStyle}
+            className='rounded-t-lg object-cover transition ease-in-out duration-300 hover:brightness-75 hover:opacity-100  '
```
Comment (ivanms1) [inline apps/web/src/components/ProjectCard/ProjectCard.tsx:81]: could you remove this extra space at the end?

**Review by ivanms1 — APPROVED** [review]
LGTM, just a minor change

File: apps/web/src/components/ProjectCard/ProjectCard.tsx (hunk 1 above)
Comment (rohilpinto) [reply apps/web/src/components/ProjectCard/ProjectCard.tsx:81]: sure

**Pull request merged**